### PR TITLE
Changing maintainer of `{rextendr}`

### PR DESCRIPTION
**Pull request by Ilia-Kosenkov**
This PR formalizes the transition of maintainership. 

@clauswilke 
@andy-thomason 
@yutannihilation

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # rextendr (development version)
 
+* Claus Wilke is stepping down as a formal maintainer. Ilia Kosenkov is accepting the role of maintainer.
```
Comment (yutannihilation) [inline NEWS.md:3]: What about simpler one like "Ilia Kosenkov is now the formal maintainer."?

e.g. https://github.com/r-lib/vroom/blob/main/NEWS.md#vroom-157

**Review by yutannihilation — APPROVED** [review]

File: NEWS.md (hunk 1 above)
Comment (clauswilke) [reply NEWS.md:3]: Maybe "Ilia Kosenkov is now the official maintainer"?

It should definitely not say "stepping down as *a* formal maintainer", as there can be only one at a time, so it's "*the* formal maintainer".

**Review by clauswilke — APPROVED** [review]
Thanks for taking this on, @Ilia-Kosenkov!

File: NEWS.md (hunk 1 above)
Comment (Ilia-Kosenkov) [reply NEWS.md:3]: You are probably correct, I did not check how it is usually written in R packages. I'll replace it with what you think is the best option. If `{vroom}` example is OK, I can adopt that.

Comment (Ilia-Kosenkov) [reply NEWS.md]: I used the phrase you both kindly suggested.

**Comment by yutannihilation** [comment]
@Ilia-Kosenkov
I think you can merge this anytime you want.

**Pull request merged**